### PR TITLE
Add AWS Rekognition Plugin

### PR DESCRIPTION
**Pull request by holidayworking**
I created AWS Rekognition Plugin.

### CODE REVIEW

File: mackerel-plugin-aws-rekognition/lib/aws-rekognition.go [hunk 1]
```diff
@@ -0,0 +1,167 @@
+package mpawswaf
```
Comment (astj) [inline mackerel-plugin-aws-rekognition/lib/aws-rekognition.go]: Please change this package name following  [Travis failure report](https://travis-ci.org/mackerelio/mackerel-agent-plugins/builds/217702224#L1179). `mpawswaf` is used by plugin-aws-waf.

File: mackerel-plugin-aws-rekognition/lib/aws-rekognition.go [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+	p.CloudWatch = cloudwatch.New(sess, config)
+
+	return nil
+}
+
+func (p RekognitionPlugin) getLastPoint(metricName string) (float64, error) {
+	now := time.Now()
+
+	response, err := p.CloudWatch.GetMetricStatistics(&cloudwatch.GetMetricStatisticsInput{
```
Comment (astj) [inline mackerel-plugin-aws-rekognition/lib/aws-rekognition.go]: According to [CloudWatch Reference](http://docs.aws.amazon.com/rekognition/latest/dg/cloudwatch-metricsdim.html), Rekognition metrics can take dimensions "Operations" optionally. Is it useful if this plugin supports operation dimensions? (like recording RequestCount metrics for each operation.)

> To retrieve operation-specific metrics, use the Rekognition namespace and provide an operation dimension.

File: mackerel-plugin-aws-rekognition/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+mackerel-plugin-aws-rekognition
+=======================
+
+AWS Rekognition custom metrics plugin for mackerel.io agent.
+
+## Synopsis
+
+```shell
+mackerel-plugin-aws-rekognition [-region=<aws-region>] -access-key-id=<id>] [-secret-access-key=<key>] [-tempfile=<tempfile>]
```
Comment (astj) [inline mackerel-plugin-aws-rekognition/README.md]: [nit]
Missing left `[` before `-access-key-id` ?

**Comment by astj** [comment]
Thank you for the p-r. Reviewed!

File: mackerel-plugin-aws-rekognition/lib/aws-rekognition.go (hunk 1 above)
Comment (holidayworking) [inline mackerel-plugin-aws-rekognition/lib/aws-rekognition.go]: Fix 795577b5d8f70cc342f6aaf8408b1e2a180c173d

File: mackerel-plugin-aws-rekognition/lib/aws-rekognition.go (hunk 2 above)
Comment (holidayworking) [inline mackerel-plugin-aws-rekognition/lib/aws-rekognition.go]: OK.
Fix 0540b774c29c9ed5c3ebc1cfbe1fa8d50b02dce9

File: mackerel-plugin-aws-rekognition/README.md (hunk 3 above)
Comment (holidayworking) [inline mackerel-plugin-aws-rekognition/README.md]: Fix 0540b774c29c9ed5c3ebc1cfbe1fa8d50b02dce9

**Comment by astj** [comment]
LGTM. Thanks!

**Comment by holidayworking** [comment]
Thanks!

**Pull request merged**